tests/TranslationTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+namespace Vanderbilt\APISyncExternalModule;
+
+class TranslationTest extends BaseTest{
```

Comment:
These tests look easy to follow and very effective!

Reply:
Great. :)